src/magic_folder/config.py

```diff
@@ -988,7 +989,7 @@ def delete_localsnapshot(self, cursor, name):
 
 
     @with_cursor
-    def store_uploaded_snapshot(self, cursor, name, remote_snapshot):
+    def store_uploaded_snapshot(self, cursor, name, remote_snapshot, upload_started_at):
```

Comment:
We also need to adjust `store_downloaded_snapshot` as well (at least to null it out). I really think, though, that this should be in a table tied to the remote snapshot, *not* in the current snapshot table.

Reply:
It should not be nulled out; the last upload happened still. We're just remembering how long that took.

Perhaps we also want to remember how long the last download took, too (since it's highly likely to be different)...

Reply:
It will not be the upload duration of the current remote-snapshot, which is what the table it is currently in is tracking. It is incorrect to keep the upload duration *in this table* if we change the remote snapshot in the table.

That is why I think we should move it to a different table, because then we can keep the data tied to the right snapshot.

Reply:
What table?

Reply:
It would need to be a new table, since we don't have a table that record remote snapshots, other than the pointer to the current one.

Reply:
Okay I nulled it out. I think making an entire snapshot-caching system is beyond the point of the immediate ask of #523